frontend/app/src/components/home/MessageEntry.svelte

```diff
@@ -285,6 +295,24 @@
      * * /details - opens group details (not yet)
      */
     function parseCommands(txt: string): boolean {
+        if (spookyTime(new Date())) {
+            const halloween = txt.match(
+                /pumpkin|zombie|skeleton|cauldron|halloween|witch|spooky|ghost/i
```

Comment:
spider, web (that might get hit more often!), creepy, scary, afraid, dark, grave, ...

Reply:
https://www.enchantedlearning.com/wordlist/halloween.shtml

Reply:
I was trying to strike a balance and not have words that are _too_ common. I think ghost is the one that guarantees that it will be discovered.

Reply:
Ok - I thought we want it to be hit quite often